libs/features/components/cards/src/lib/pages/elewa-group-card/elewa-group-card.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="row row-cols-1 row-cols-md-3 g-4">
+    <div class="col">
+        <div class="card h-100">
+            <h5 class="card-subtitle"></h5>
+            <h2 class="card-title"></h2>
+            <img src="https://unsplash.com/photos/IgUR1iX0mqM" class="card-img-center" alt="woman and man sitting infront of a computer">
+            <p class="card-text"></p>
+        </div>
+    </div>
+    <div class="col">
+        <div class="card h-100">
+            <h5 class="card-subtitle"></h5>
+            <h2 class="card-title"></h2>
+            <img src="https://unsplash.com/photos/HOrhCnQsxnQ" class="card-img-center" alt="">
+            <p class="card-text"></p>
+        </div>
+    </div>
+    <div class="col">
+        <div class="card h-100">
+            <h5 class="card-subtitle"></h5>
+            <h2 class="card-title"></h2>
+            <img src="" class="card-img-center" alt="">
```

Comment:
There's no code for changing image placement, read issue description.

Reply:
Hello, I have modified the code to include a code for changing image placement and pushed my changes for review

Reply:
This code here for card component am not working on it anymore. I updated my repo wiht the merged work.